### PR TITLE
Localize solar eclipse file name

### PR DESCRIPTION
**Pull request by 10110111**
The default output file name for eclipse maps will now depend on locale.

### CODE REVIEW

File: src/gui/AstroCalcDialog.cpp [hunk 1]
```diff
@@ -5043,18 +5043,17 @@ void AstroCalcDialog::saveSolarEclipseMap()
...
-	int year, month, day;
-	StelUtils::getDateFromJulianDay(JDMid, &year, &month, &day);
-
 	core->setUseTopocentricCoordinates(savedTopocentric);
 	core->update(0);
 
 	// Use year-month-day in the file name
-	const auto eclipseDateStr = QString("-%1-%2-%3").arg(year).arg(month,2,10,QChar('0')).arg(day,2,10,QChar('0'));
+	const auto eclipseDateStr = localeMgr->getPrintableDateLocal(JDMid);
+	const auto fileBaseName = qc_("Solar eclipse %1",
```
Comment (alex-w) [inline src/gui/AstroCalcDialog.cpp:5051]: The context data is very long. Maybe just comment for translators and short context (or without context)

Comment (10110111) [reply src/gui/AstroCalcDialog.cpp:5051]: How would you formulate this in a shorter way, keeping it unambiguous?

Comment (10110111) [reply src/gui/AstroCalcDialog.cpp]: OK, I've dropped this, since the simple concatenation is already used in the caption inside KML.

**Review by alex-w — APPROVED** [review]
Thanks!

**Pull request merged**